src/Files.App/Utils/Storage/StorageItems/NativeStorageFile.cs

```diff
@@ -105,12 +105,14 @@ private void CreateFile()
 
 		public override IAsyncAction DeleteAsync()
 		{
-			return AsyncInfo.Run(async (cancellationToken) =>
+			return AsyncInfo.Run(_ =>
```

Comment:
Is this change correct?

Reply:
Yes, the cancellation token is unused and the method don't have any `await` usage. With this change no state machine is generated.